HW3/Program.cs

```diff
@@ -0,0 +1,122 @@
+﻿using HW3.Exceptions;
+using System;
+using System.Collections.Generic;
+using System.IO;
+
+namespace HW3
+{
+    internal class Program
+    {
+        private const string usersFile = "Users.txt";
+
+        private static bool ConsoleLogin()
+        {
+            try
+            {
+                string username = ConsoleReadLineWithTextPrompt("Username:\t");
```

Comment:
A method name does not have to have all those details. Simply call it: `PromptText`